src/App.svelte

```diff
@@ -817,4 +343,4 @@
       Discord</a
     >
   </div>
-</div>
+</div>
```

Comment:
could you add an empty line at the end? An empty line removes this 😃 :
![imagem](https://user-images.githubusercontent.com/18630253/94373857-1e01e300-0100-11eb-99a8-5f2e2fa31f15.png)